website/materials.html

```diff
@@ -0,0 +1,111 @@
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-white" href="about.html">About</a></li>
+      <li class="nav-item">
+        <a class="nav-link text-white" href="materials.html">Materials</a> </li>        <li class="nav-item dropdown">
+       <a class="nav-link dropdown-toggle text-white" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Tutorials
+          </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+            <a class="dropdown-item text-black" href="hardware.html">Telescope Design</a>
+            <a class="dropdown-item text-black" href="software.html">Software Setup</a>
+            <a class="dropdown-item text-black" href="datataking.html">Taking data</a>
+            <a class="dropdown-item text-black" href="analysis.html">Analysis</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="faq.html">FAQ</a> </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="contact.html">Contact</a> </li>
+        </ul>
+</div>
+</div>
+</nav>
+<h2 id="centerheader" style= "border:solid;border-color:black;" font-family:"courier new", courier, arial; >Suggested Materials</h2>
+</header>
+
+
+
+  <div class="container-fixed" style="padding:0px; margin:0px;">
+  <section class="section">
+  <p style="margin:20px;"> This list is intended to serve as a starting place for some suggested materials to build a version of CHART.
+    
+    <br> Materials for building:
+    <br>-4 36” x 48” Double walled cardboard sheets.
+    <br>-75 square feet: Aluminum Foil.
+    <br>-Hot Glue Gun with Hot Glue.
+    <br>-Tape.
+    <br>-Aluminum Foil Tape.
+    <br>-Raspberry Pi 3B
+    <br>-Micro SD Card (32GB)
+    <br>-RTL-SDR
+    <br>-A computer monitor
+    <br>-Ethernet cable (if not using WiFi option)
+    <br>-Keyboard
+    <br>-Personal computer
+    <br>-2 Gold plated nuts.
+    <br>-1 Gold plated locking washer.
+    <br>-1 small scrap aluminum (a soup can lid covered in aluminum tape will work fine)
+    <br>-1 Coaxial Panel Mount Connector with Bulkhead Nut and Solder Cup Terminal SMA Male to Female Right Angle 90-Degree Adapter Gold Plated.
+    <br>-2 Low noise amplifiers (LNA)
+    <br>-2 batteries for LNA (check spec voltage)
+    <br>-2 battery pack for soldering onto LNA
+    <br>-6.3cm Copper wire.
+    <br>-SMA style terminator 
+    <br>-Long SMA cable (enough to run from monitor to wherever you want to take data) 
+
+    <br>
+    <br>Reccomended extras:
+    <br>Bandpass filter for 21cm
+    <br>Multimeter with continuity functionality 
+    <br>Soldering kit 
+    <br>Small radio antenna 
+
+
+
+  <div class="pimg1">
+    <div class="ptext">
+      <span class="border">
+        Completely Hackable Amateur Radio Telescope
+      </span>
+      </div>
+  </div>
```

Comment:
I don't think you want this. It makes a banner that blocks the bottom of the page.